finalPrep/README.md

```diff
@@ -31,24 +31,26 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+My calendar is a bit incomplete and starts at the wrong cohort (I deferred to 2111), but that was the only calendar available through the link provided.
+- ![Screen Shot 2021-09-26 at 1 10 35 AM](https://user-images.githubusercontent.com/90285348/134799191-a47d7f1a-a489-4eef-bf38-f5308b6ec36b.png)
+- ![Screen Shot 2021-09-26 at 1 10 45 AM](https://user-images.githubusercontent.com/90285348/134799188-ac0ab693-051a-41ec-a8cd-878e945aef59.png)
+- ![Screen Shot 2021-09-26 at 1 10 56 AM](https://user-images.githubusercontent.com/90285348/134799190-954c615c-0f99-4a04-89ab-580fba4e23a1.png)
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: https://gist.github.com/Prkr93/17d7bc3888206ef4b5752024c2a79071
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: https://gist.github.com/Prkr93/04b993d6469c40299e743f79155b93de
```

Comment:
Nice start with notes. I'd focus on figuring out a system for using colors, like putting vocab terms in a particular color, code examples in a particular color.